### DISCUSSION TITLE
Border Radius: Inherit

### DISCUSSION THREAD

**Discussion by mnaderian** [tailwindlabs/tailwindcss#5348] [Ideas]
Is it ok to ask if possible to add the `inherit` value for the border radius like `rounded-inherit`? 
And also other possible utilities for top, bottom, left and right.

Thanks,
And love you all guys!

**Comment by jjdevzinho** [comment]
It would be great to have the "inherit" for the rounded classes

**Reply by luizov** [reply]
Just stumbled upon this myself. For anyone interested, you can now do `rounded-[inherit]` with arbitrary values and it works wonderfully. Here is an example with an ::after pseudo element inheriting its border radius: [https://play.tailwindcss.com/ZZ6Pg9y79H](https://play.tailwindcss.com/ZZ6Pg9y79H)

**Comment by gregg-cbs** [comment]
How would we add this to tailwind config so we create a rounded-inherit class?

I tried:
```js
 extend: {
  rounded: {
    inherit: "inherit"
  }
 }
```

**Reply by gregg-cbs** [reply]
Ahhh because sometimes naming is confusing, this is the answer:

```js
 extend: {
  borderRadius: {
    inherit: "inherit"
  }
 }
```

**Reply by ivands** [reply]
How would this work in v4 tho?

**Reply by wongjn** [reply]
```css
@theme inline {
  --radius-inherit: inherit;
}
```
https://play.tailwindcss.com/GOatjyFabg

The `inline` tag is needed so that the value is `inherit` in the class rule, not `var(--radius-inherit)`. The former correctly inherits from the parent, the later would inherit from `:root`'s parent (nothing) and thus be nothing before evaluating "down".

**Comment by makschernetskyi** [comment]
class="rounded-[inherit]"
